docs/docs/cmd/spo/list/list-roleinheritance-break.md

```diff
@@ -0,0 +1,51 @@
+# spo list roleinheritance break
+
+Break inheritance on list or library. Keeping existing permissions is the default behavior.
+
+## Usage
+
+```sh
+m365 spo list roleinheritance break [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the list to retrieve is located
+
+`-i, --listId [listId]`
+: ID of the list to retrieve information for. Specify either id or title but not both
+
+`-t, --listTitle [listTitle]`
+: Title of the list to retrieve information for. Specify either id or title but not both
+
+`-c --clearExistingPermissions`
```

Comment:
Let's separate the short and long option with a comma

Reply:
done